examples/bevy/src/main.rs

```diff
@@ -29,15 +29,14 @@ fn main() {
         .build()
         .unwrap();
 
-    // Next, we make an executor, loading the ELF binary.
-    let mut exec = default_executor_from_elf(env, BEVY_GUEST_ELF).unwrap();
+    // Obtain the default prover.
+    let prover = default_prover();
 
-    // Run the executor to produce a session.
-    let session = exec.run().unwrap();
-
-    // Prove the session to produce a receipt.
-    let receipt = session.prove().unwrap();
+    // Produce a receipt by proving the specified ELF binary.
+    let receipt = prover.prove_elf(env, BEVY_GUEST_ELF).unwrap();
 
+    // The prover already runs a verify internally, but this is how other users
```

Comment:
hmm... this comment feels odd to me. mb this is better?
```suggestion
    // After running a user generates a receipt, other users can verify the receipt using: 
```

Reply:
is there an easy way to suggest a revision on github that spans multiple lines?

(i.e., if you hit commit on this it will leave line 39 hanging and that's annoying)

Reply:
When selecting the line to comment on, click and drag to select multiple lines to comment on. This works even if you don't suggest revisions, but if you do suggest revisions it lets you revise all the selected lines.

(see my multiline comment for my thoughts on the actual comment here)